server/src/schema_config_provider/cluster_based.rs

```diff
@@ -4,22 +4,28 @@
 
 #![allow(dead_code)]
 
-use cluster::ClusterRef;
+use cluster::{topology::SchemaConfig, ClusterRef};
 
 use crate::schema_config_provider::{Result, SchemaConfigProvider};
 
 pub struct ClusterBasedProvider {
     cluster: ClusterRef,
+    default_schema_config: SchemaConfig,
```

Comment:
According to the `FIXME` beneath this `default_schema_config` is also temporary, right?

Reply:
Yes. Now the schema information is not supported to be stored by CeresMeta. The schema configurations will be fetched from `Cluster` in the future.